.node-version

```diff
@@ -0,0 +1 @@
+^8.1.0
```

Comment:
This file is used by AVN to automatically switch to the correct node version. You won't need this, but you should use NVM to install node 8 instead of node 6 which you're likely currently on.